apps/web/src/services/routes/api.ts

```diff
@@ -0,0 +1,41 @@
+export const _API_ROUTES = {
+  documentLogs: {
+    detail: ({ id }: { id: number }) => ({
+      root: `/api/documentLogs/${id}`,
+    }),
+    uuids: {
+      detail: ({ uuid }: { uuid: string }) => ({
+        root: `/api/documentLogs/uuids/${uuid}`,
+      }),
+    },
+  },
+  documents: {
+    detail: ({ projectId }: { projectId: number }) => {
+      const documentsAtProject = `/api/documents/${projectId}`
+      return {
+        detail: ({ commitUuid }: { commitUuid: string }) => {
+          const documentsAtCommit = `${documentsAtProject}/${commitUuid}`
+          return {
+            root: documentsAtCommit,
+            detail: ({ documentUuid }: { documentUuid: string }) => {
+              const documentRoot = `${documentsAtCommit}/${documentUuid}`
+              return {
+                evaluations: {
+                  root: `${documentRoot}/evaluations`,
+                },
+                evaluationResultsByDocumentContent: {
+                  detail: ({ evaluationId }: { evaluationId: number }) => ({
+                    root: `${documentRoot}/evaluation-results-by-document-content/${evaluationId}`,
+                  }),
+                },
+              }
+            },
+          }
+        },
+        forImport: {
+          root: `${documentsAtProject}/for-import`,
+        },
+      }
+    },
+  },
+}
```

Comment:
usually not the best idea to do these kind of tangential refactors when you are in a mega ultra massive PR already, a tip for next times :)

Reply:
This was asked for in another smaller PR. This PR is just the whole trenecito ready to merge. It was split into several smaller ones for easier review.